decidim/decidim
# Configurable default order for proposals

#### :tophat: What? Why?
Add a setting to the proposal component that specifies the default order of the proposals. Admin can also set different default order for each phase.

#### :pushpin: Related Issues
https://meta.decidim.org/processes/roadmap/f/122/proposals/13104

#### Testing
1. Login as admin -> Admin Dashboard -> Edit component
2. Change "Default proposal sorting" to something else than default. (If you want test "most endorsed" for example, make sure you have endorsements enabled).
3. Go to proposals index view and check "Order proposals by"

#### :clipboard: Checklist
:rotating_light: Please review the [guidelines for contributing](https://github.com/decidim/decidim/blob/develop/CONTRIBUTING.adoc) to this repository.


### :camera: Screenshots
![image](https://user-images.githubusercontent.com/19709320/131997078-1c629b0d-c51c-4c9e-a2da-50680c92f044.png)

![image](https://user-images.githubusercontent.com/19709320/131997648-6ce54c65-ef2b-42b0-8c51-cec54b6425b2.png)

:hearts: Thank you!
